okio/src/commonTest/kotlin/okio/util.kt

```diff
@@ -33,6 +34,17 @@ fun segmentSizes(buffer: Buffer): List<Int> {
   return sizes
 }
 
+fun assertByteArraysEquals(a: ByteArray, b: ByteArray) {
```

Comment:
```suggestion
fun assertArrayEquals(a: ByteArray, b: ByteArray) {
```

Matches JUnit. I think I'll send a Kotlin change to add these methods to kotlin.test...

Reply:
https://youtrack.jetbrains.com/issue/KT-32996